### PR TITLE
Add Elasticsearch image to CR and flag

### LINKED ISSUE #215: Add flag and CR option for setting elasticsearch image
The question is whether we need it as the image is configurable on ES operator -https://github.com/openshift/elasticsearch-operator#openshift-alternatives---make-targets

At the moment the image is not configurable and we don't set it on ES CR. The ES operator will set it.

### PR DESCRIPTION
**Pull request by pavolloffay**
Resolves #215 

Signed-off-by: Pavol Loffay <ploffay@redhat.com>

### CODE REVIEW

**Comment by jpkrohling** [comment]
This change is [Reviewable](https://reviewable.io/reviews/jaegertracing/jaeger-operator/289)

File: pkg/cmd/start/main.go [hunk 1]
```diff
@@ -66,6 +66,9 @@ func NewStartCommand() *cobra.Command {
 	cmd.Flags().String("jaeger-es-rollover-image", "jaegertracing/jaeger-es-rollover", "The Docker image for the Jaeger Elasticsearch Rollover")
 	viper.BindPFlag("jaeger-es-rollover-image", cmd.Flags().Lookup("jaeger-es-rollover-image"))
 
+	cmd.Flags().String("jaeger-elasticsearch-image", "quay.io/openshift/origin-logging-elasticsearch5:latest", "The Docker image for Elasticsearch")
```
Comment (pavolloffay) [inline pkg/cmd/start/main.go]: https://github.com/openshift/elasticsearch-operator/blob/master/hack/cr.yaml#L9

**Comment by pavolloffay** [comment]
Not sure why it generates two additional files - @jpkrohling shall we ignore them?

**Comment by jpkrohling** [comment]
@pavolloffay those are new as of 0.5.0. You can leave them in this PR, just include them in the `.codecov.yml` file, so that they aren't counted for the coverage. Or you can remove them from this PR and I'll include them in a follow-up PR.

**Pull request merged**